cmd/collector/app/handler/grpc_handler.go

```diff
@@ -58,12 +58,12 @@ type batchConsumer struct {
 	logger        *zap.Logger
 	spanProcessor processor.SpanProcessor
 	spanOptions   processor.SpansOptions
-	tenancyConfig tenancy.TenancyConfig
+	tenancyMgr    *tenancy.TenancyManager
 }
 
-func newBatchConsumer(logger *zap.Logger, spanProcessor processor.SpanProcessor, transport processor.InboundTransport, spanFormat processor.SpanFormat, tenancyConfig *tenancy.TenancyConfig) batchConsumer {
-	if tenancyConfig == nil {
-		tenancyConfig = &tenancy.TenancyConfig{}
+func newBatchConsumer(logger *zap.Logger, spanProcessor processor.SpanProcessor, transport processor.InboundTransport, spanFormat processor.SpanFormat, tenancyMgr *tenancy.TenancyManager) batchConsumer {
+	if tenancyMgr == nil {
```

Comment:
This line only became part of the PR because of the rename from `TenancyConfig` to `TenancyManager`.  It was already merged this way.  _oltp_receiver.go_:146 actually does pass in nil.  My thought at that time was to default to an empty handler.  If you would prefer it to be explicit I will also add an `error` return and fail if `nil` is passed in.  Thoughts?